### ISSUE TITLE
Qute is not adding the right NativeImageResourceBuildItem when using a custom template root

### ISSUE THREAD

**Issue by ia3andy** [quarkusio/quarkus#39197]
### Describe the bug

Using `web/templates` as a template root with `TemplateRootBuildItem`, it's not adding the right `NativeImageResourceBuildItem`

See:
```java
2024-03-05 18:10:10,609 DEBUG [io.qua.qut.dep.QuteProcessor] (build-32) Produce template build items [filePath: WebResource/page1.html, fullPath: templates/WebResource/page1.html, originalPath: /Users/adamevin/workspace/redhat/quarkiverse/quarkus-web-bundler/integration-tests/target/classes/web/templates/WebResource/page1.html
```

in https://github.com/quarkusio/quarkus/blob/main/extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/QuteProcessor.java#L3393

it's adding the `fullPath` in the `NativeImageResourceBuildItem` which is `templates/WebResource/page1.html` it should be `web/templates/WebResource/page1.html`

### Quarkus version or git rev

3.8.1

This is what I mentioned in https://github.com/quarkiverse/quarkus-web-bundler/pull/165#pullrequestreview-1888549715

**Comment by ia3andy** [comment]
I guess the issue is also the same for the `HotDeploymentWatchedFileBuildItem`

**Comment by mkouba** [comment]
@ia3andy this wasn't fixed by quarkiverse/quarkus-web-bundler#174, right?

**Comment by ia3andy** [comment]
@mkouba nop, I am not sure why it was closed.. for now there is no way to work around it properly from the Web Bundler

**Comment by ia3andy** [comment]
Seriously me mentioning the issue closed it 😱 

> Using web/templates for Qute templates will not work in native until this gets fixed: https://github.com/quarkusio/quarkus/issues/39197

**Comment by ia3andy** [comment]
And this issue doesn't want to be fixed it seems as @cescoffier mentioned it also as fixed by mistake 🤣

**Comment by mkouba** [comment]
> Seriously me mentioning the issue closed it 😱
> 
> > Using web/templates for Qute templates will not work in native until this gets fixed: #39197

Yep, `fixed: issueId` is IMO one of the phrases that triggers closing.

**Comment by mkouba** [comment]
@ia3andy Just to double-check - in your case a `TemplateRootBuildItem` with the path `web/templates` is produced, right?
